portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,9 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+// testing function for step 2
```

Comment:
Comments should be for the reader/reviewer, in this case I'm not sure what the context is here.